doc: a small facelift (#7)
mloncode/amld2019-workshop: README.md

- [gitbase](https://docs.sourced.tech/gitbase)
- [bblfsh](https://doc.bblf.sh)
- [BigARTM](http://bigartm.org)

> - [PyTourch](https://pytorch.org)

Reviewer: ```suggestion
- [PyTorch](https://pytorch.org)
```

Author: 🤦‍♂ tnx!

 <details>
<summary>Abstract</summary>

> > Machine Learning on Source Code (MLonCode) is an emerging research domain which stands at the > intersection of deep learning, natural language processing, software engineering and programming > language communities.

Reviewer: ```suggestion
> Machine Learning on Source Code (MLonCode) is an emerging research domain which stands at the intersection of deep learning, natural language processing, software engineering and programming language communities.
```

>

> > During this 3.5 hours workshop, we will review the recent SE tasks that benefit from applying ML and focus the hands-on experience on:

Reviewer: ```suggestion
> During this 3h30 workshop, we will review recent Software Engineering tasks that benefit from applying Machine Learning, with a focus on hands-on experience on:
```

> > - extracting data from the real source code and

Reviewer: ```suggestion
> - extracting data from real source code
```

> > - developing multiple different ML models

Reviewer: ```suggestion
> - developing multiple Machine Learning models
```

> > - for a particular task of source code summarization (or function name suggestion).

Reviewer: ```suggestion
> - summarizing source code, suggesting function names and analyzing projects and developers most relevant topics
```

Author: So far this text was 1:1 copy of the materials published on the website.
I'm not sure what's best to do in here

Reviewer: It's probably fine to improve it here but I'm also fine keeping it as is.

>

> > At the end of the workshop participants will build 2 working models on a real dataset, producing > near state-of-the-art results. Practical skill of extracting information from source code as well > as modeling different aspects of it are going to be acquired.

Reviewer: ```suggestion
> At the end of the workshop participants will build 2 working models on a real dataset, producing near state-of-the-art results. Practical skill of extracting information from source code as well as modelling different aspects of it are going to be acquired.
```
